.github/workflows/build.yml

```diff
@@ -805,20 +804,3 @@ jobs:
       - name: Build LLVM ${{ matrix.llvm }} on ${{ matrix.ubuntu }}
         run: |
           docker build . -t docker.pkg.github.com/lifting-bits/anvill/anvill-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest -f Dockerfile --target=dist --build-arg REMILL_COMMIT_ID=${REMILL_COMMIT_ID} --build-arg UBUNTU_VERSION=${{ matrix.ubuntu }} --build-arg ARCH=amd64 --build-arg LLVM_VERSION=${{ matrix.llvm }}
-      - name: Test Docker image
```

Comment:
This seems to be more a consequence of migrating to the Protobuf format right? Let's have that as a separate change.